airflow/decorators/__init__.pyi

```diff
@@ -269,6 +287,8 @@ class TaskDecoratorFactory:
     @overload
     def python(self, python_callable: F) -> F: ...
     @overload
+    def branch(self, python_callable: F) -> F: ...
+    @overload
```

Comment:
Ah good -- you spotted my change, this will need moving to be next to the other branch.

Reply:
@subkanthi My PR is merged now.

Reply:
Changed, thanks